common/src/main/java/gov/cms/ab2d/common/repository/CoverageRepository.java

```diff
@@ -22,4 +23,28 @@
 
     Optional<Coverage> findByContractAndBeneficiaryAndPartDMonth(Contract contract, Beneficiary bene, int month);
 
+
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  ")
+    int deleteAllInBulk();
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  " +
+            " WHERE c.partDMonth = :month  ")
+    int deleteInBulk(int month);
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  " +
+            " WHERE c.contract.id = :contractId  ")
+    int deleteInBulk(Long contractId);
+
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  " +
+            " WHERE c.contract.id = :contractId  " +
+            "   AND c.partDMonth = :month  ")
+    int deleteInBulk(Long contractId, int month);
```

Comment:
...following up on the above, `deleteByMonthAndContract` would settle this interface.

Reply:
done